apiserver/common/networkingcommon/networkconfigapi.go

```diff
@@ -8,10 +8,13 @@ package networkingcommon
 
 import (
 	"net"
+	"strings"
 
+	"github.com/juju/collections/set"
 	"github.com/juju/errors"
 	"github.com/juju/loggo"
 	"github.com/juju/names/v4"
+	"gopkg.in/mgo.v2/txn"
```

Comment:
While I get that this is a `common` package and why the *Op() methods are in here, it's quite concerning to have low-level db-specific details in any sub-package of `apiserver`. That's probably something we should be actively working to change as part of future refactoring work.

Reply:
I see your point. This is a hard one - composing model operations is intended to get the business logic out of the state package, but we leak the `txn.Op` detail.

Reply:
I agree with @achilleasa on this, leaking `txn.Op` is really bad here.